src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  background-repeat: no-repeat;
+
+  &__star {
+    @include star-size (15px);
+    margin-right: 4px;
+  
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+  }
+
+  & :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
You could do this not only for this case, but for all `stars--` options from 1 to 5 using SCSS loop